docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -2133,6 +2133,7 @@ Display system-wide information
 
     {
         "Architecture": "x86_64",
+        "ClusterStore": "etcd://localhost:2379",
```

Comment:
Looks like we need to update this in the 1.21 and 1.23 docs as well https://github.com/docker/docker/search?utf8=✓&q=DiscoveryBackend